tests/unit/script/lib.rs

```diff
@@ -7,6 +7,7 @@
 #[cfg(test)] extern crate script;
 #[cfg(test)] extern crate servo_url;
 
+#[cfg(test)] mod dom;
```

Comment:
Just make a top-level module named `str`, no need to follow the module hierarchy of the script crate.